### DISCUSSION TITLE
Migration from Zotero and issue with not papis not finding files

### DISCUSSION THREAD

**Discussion by Archie-2021** [papis/papis#866] [Q&A]
Hey,

I recently found this amazing tool , and trying to move my library to papis.

In doing that I am trying to divid my big library into some smaller ones (because it is so easy to move files between libraries. But I got into a strange issue. On library is frozen to only one entry/document while when I copy the exact same files to another library papis can read it perfectly well. And both libraries are defined in my config file. I tried update command but has no effect. 

This is a really cool application, the only issue it looks lack of information about it. I mean documentation is really decent but walking through and getting used a new tool is much more than just reading documentation. I really liking this application and hope to find a source to find my way in it.

Thanks

**Comment by jghauser** [comment]
Hi and welcome to Papis! 

One possibility is that the cache isn't being updated? you can do so with `papis cache clear` (or `papis --cc` if you're on an older version of Papis).

I agree with your comments regarding the documentation, especially when it comes to easy-to-follow introductions. We're currently rewriting some of it, you might find #827 helpful :).

**Reply by Archie-2021** [reply]
> Hi and welcome to Papis!
> 
> One possibility is that the cache isn't being updated? you can do so with `papis cache clear` (or `papis --cc` if you're on an older version of Papis).
> 
> I agree with your comments regarding the documentation, especially when it comes to easy-to-follow introductions. We're currently rewriting some of it, you might find #827 helpful :).

Hi , and thank for the help

Sorry I missed the notification so my later reply.

I got fixed it with `papis --cc` command. It is weird because both libraries got updated at the same time.

I am using 0.13 which I got from pip.

**Comment by alexfikl** [comment]
To follow up on @jghauser's suggestions, can you give more details about the problem you're facing? From the top of my heads, these could help narrow down the issue:

* What version of `papis` are you using? There have been a lot of changes since 0.13, so maybe you can try the git version :grin:
* What commands are you running to import the zotero library? Did they give any errors?
* What errors are you getting for which commands?
* If you have multiple little libraries, you'll need to say which one you want to query with `papis -l MY_LIBRARY open SOME_QUERY`. What commands are you running to look for files?

**Reply by Archie-2021** [reply]
> To follow up on @jghauser's suggestions, can you give more details about the problem you're facing? From the top of my heads, these could help narrow down the issue:
> 
> * What version of `papis` are you using? There have been a lot of changes since 0.13, so maybe you can try the git version 😁
> * What commands are you running to import the zotero library? Did they give any errors?
> * What errors are you getting for which commands?
> * If you have multiple little libraries, you'll need to say which one you want to query with `papis -l MY_LIBRARY open SOME_QUERY`. What commands are you running to look for files?

I am using 0.13 which I got from pip. I wonder how outdated it is as I see many commands. How outdated is it? I am a bit hesitant to compile programs as it can make a mess on long run. But I will make a exception if the gap between package updates and git version is a lot. I usually use AUR for this , but it is one of the rare occasions that I ran into problems with an AUR package.

**Reply by alexfikl** [reply]
There's a (incomplete) change log here https://github.com/papis/papis/blob/c32c763f3a4b43902b321f712964279cd0238ebe/CHANGELOG.md. There are quite a few changes and a lot of smaller fixes, so it might make a difference.

I'm running the `-git` version from AUR as well, but that's mostly because it helps with development and catching bugs a bit earlier. We should release a new version sometime soon, so hopefully this won't be an issue for much longer :grin:
